samples/csharp/web/avatar/README.md

```diff
@@ -0,0 +1,103 @@
+# Instructions to run Microsoft Azure TTS Talking Avatar sample code
+
+## Pre-requisites
+
+* Follow [Text to speech quickstart](https://learn.microsoft.com/azure/ai-services/speech-service/get-started-text-to-speech?pivots=programming-language-python#set-up-the-environment) to set up the environment for running Speech SDK in python.
```

Comment:
This should be the csharp version instead of python version

Reply:
updated